aml_module/tests/test_error_input.py

```diff
@@ -6,14 +6,13 @@
 import pandas as pd
 import shutil
 import os
-from azureml.studio.core.io.data_frame_directory import save_data_frame_to_directory
 from azureml.studio.core.error import UserError
 import invoker
 
 
 class TestErrorInput(unittest.TestCase):
     def setUp(self):
-        self.__input_path = './test_input_data_frame_directory'
+        self.__input_path = '../../samples/sample_error.csv'
```

Comment:
can you mock the sample_error.csv by code? It is not suggested to put an error file in sample.